airflow-core/tests/unit/models/test_deadline.py

```diff
@@ -94,6 +94,40 @@ def test_add_deadline(self, dagrun, session):
         assert result.callback == deadline_orm.callback
         assert result.callback_kwargs == deadline_orm.callback_kwargs
 
+    def test_add_multiple_deadlines(self, dagrun, session):
```

Comment:
Isn’t think essentially just testing SQLAlchemy relationship works? I don’t think we generally do this.

Reply:
Makes sense. Thanks.